cookbooks/hangops-jobbot/recipes/nodejs.rb

```diff
@@ -51,3 +51,9 @@
           '/srv/hangops-jobbot/vendor/' \
           'hubot-slack-whitelist-middleware --save'
 end
+
+# Install hubot-rate-limit
+execute 'install-rate-limit' do
+  command 'cd /srv/hangops-jobbot;' \
+  'npm install hubot-rate-limit --save'
+end
```

Comment:
https://github.com/rrxtns/hangops-jobbot/blob/7ca6440df34d24a7aa774738cfe820c1641dfbd1/package.json#L13 - ```package.json``` has it.